src/v/pandaproxy/json/iobuf.h

```diff
@@ -84,7 +84,6 @@ class rjson_serialize_impl<iobuf> {
         if (buf.empty()) {
             return w.Null();
         }
-        // TODO Ben: Implement custom OutputStream to prevent this linearization
         return w.String(iobuf_to_base64(buf));
```

Comment:
Think this commit is worth backporting?

Reply:
Meh, I don't think pandaproxy has any real world usage and this code is super old. I am inclined to not do so.